sims/vsim/Makefile

```diff
@@ -75,14 +74,22 @@ VCS_NONCC_OPTS = \
 
 VCS_OPTS = -notice -line $(VCS_CC_OPTS) $(VCS_NONCC_OPTS)
 
+########################################################################################
+# remove duplicate blackboxes
+########################################################################################
+sim_vcs_blackboxes ?= $(build_dir)/firrtl_black_box_resource_files.vcs.f
+
+$(sim_vcs_blackboxes): $(sim_top_blackboxes) $(sim_harness_blackboxes)
+	join -a 1 -a 2 <(sort $(sim_top_blackboxes)) <(sort $(sim_harness_blackboxes)) > $@
```

Comment:
There's a much simpler way to do this that doesn't require parallel pipes.

```
cat $^ | sort | uniq > $@
```

Reply:
LOL, I guess that doesn't work because the blackbox files don't have ending newlines.

Reply:
You can use `awk '{print $1;}'` to get around the trailing newline issue. Also, `sort -u` would remove the extra `| uniq`, so

```
awk '{print $1;}' $^ | sort -u > $@
```

should work.

Reply:
@abejgonzalez I confirmed this worked, it's a bit cleaner than the anonymous pipes.

Reply:
Nice